src/SlackMessage.mock.js

```diff
@@ -8,30 +8,30 @@ class SlackMessage extends SlackMessageRoot {
 
   constructor(ts) {
     super();
-    this._mockTs = ts;
-    this._spies = {
+    this.#_mockTs = ts;
+    this.#_spies = {
```

Comment:
The only changes in this file are to use the hashtag-prefixed instance variables. It looks like the original author of the class intended to use JS private class features but omitted the `#` inside the actual class.